src/test/java/com/atlassian/jira/cloud/jenkins/listeners/AutoBuildsAndDeploymentsTest.java

```diff
@@ -192,24 +194,23 @@ public void whenNoAutoBuildsRegex_thenSendsInProgressAndSuccessBuildEvents() thr
         givenIssueKeys();
 
         jenkins.assertBuildStatusSuccess(workflow.scheduleBuild2(0));
-
-        verifyBuildEvent(0, State.IN_PROGRESS);
-        verifyBuildEvent(1, State.SUCCESSFUL);
+        verifyAtLeastOneBuildEvent(State.IN_PROGRESS);
+        verifyExactlyOneBuildEvent(State.SUCCESSFUL);
```

Comment:
I had to change the logic of this test, because the CHANGE_BRANCH env var is injected into the CI build and changed the behavior so that multiple IN_PROGRESS events were sent to Jira. So I made the test look for at least one IN_PROGRESS event instead of exactly one.